examples/green-house-migration/routes/api/candidates.py

```
@@ -0,0 +1,1388 @@
+                except Exception as e:
+                    logger.warning("Failed to add candidate to prospect pool: %s", e)
+            else:
+                logger.warning("Prospect pool '%s' not found", request.prospect_pool)
+
+        return candidate
+
+    except Exception as e:
+        logger.error("Failed to create candidate: %s", e)
+        raise HTTPException(
+            status_code=500, detail="Failed to create candidate: {str(e)}"
+        )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)